### PR TITLE
docs: update references, remove docs dir

### PR DESCRIPTION
**Pull request by desimone**
## Summary

Removes the old `docs` dir which has been migrated to the documentation repo (https://github.com/pomerium/documentation) ; and updates the references to our .com domain. 

## Related issues

- https://github.com/pomerium/pomerium-console/issues/2570
- https://github.com/pomerium/documentation/issues/23
- closes https://github.com/pomerium/pomerium/pull/3419

## Checklist

- [x] reference any related issues
- [x] updated docs
- [ ] updated unit tests
- [ ] updated UPGRADING.md
- [x] add appropriate tag (`improvement` / `bug` / etc)
- [x] ready for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ For comprehensive docs, and tutorials see our [documentation].
 [documentation]: https://pomerium.com/docs/
 [go environment]: https://golang.org/doc/install
 [godocs]: https://godoc.org/github.com/pomerium/pomerium
-[quick start guide]: https://www.pomerium.io/guide/
+[quick start guide]: https://www.pomerium.com/guide/
```
Comment (alexfornuto) [inline README.md:26]:
```suggestion
[quick start guide]: https://www.pomerium.com/docs/install/quickstart
```

File: config/options.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ import (
 const DisableHeaderKey = "disable"
 
 const (
-	idpCustomScopesDocLink = "https://www.pomerium.io/reference/#identity-provider-scopes"
+	idpCustomScopesDocLink = "https://www.pomerium.com/reference/#identity-provider-scopes"
```
Comment (alexfornuto) [inline config/options.go:43]:
```suggestion
	idpCustomScopesDocLink = "https://www.pomerium.com/docs/reference/identity-provider-scopes"
```

File: examples/mutual-tls/README.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ services:
       - CERTIFICATE_KEY
       - COOKIE_SECRET
     volumes:
-      # Mount your config file : https://www.pomerium.io/reference/
+      # Mount your config file : https://www.pomerium.com/reference/
```
Comment (alexfornuto) [inline examples/mutual-tls/README.md:48]:
```suggestion
      # Mount your config file : https://www.pomerium.com/docs/reference/
```

File: examples/mutual-tls/docker-compose.yaml [hunk 4]
```diff
@@ -7,7 +7,7 @@ services:
       - CERTIFICATE_KEY
       - COOKIE_SECRET
     volumes:
-      # Mount your config file : https://www.pomerium.io/reference/
+      # Mount your config file : https://www.pomerium.com/reference/
```
Comment (alexfornuto) [inline examples/mutual-tls/docker-compose.yaml:10]:
```suggestion
      # Mount your config file : https://www.pomerium.com/docs/reference
```

File: internal/identity/manager/manager.go [hunk 5]
```diff
@@ -212,7 +212,7 @@ func (mgr *Manager) refreshDirectoryUserGroups(ctx context.Context) (nextRefresh
 		msg := "failed to refresh directory users and groups"
 		if ctx.Err() != nil {
 			msg += ". You may need to increase the identity provider directory timeout setting"
-			msg += "(https://www.pomerium.io/reference/#identity-provider-refresh-directory-settings)"
+			msg += "(https://www.pomerium.com/reference/#identity-provider-refresh-directory-settings)"
```
Comment (alexfornuto) [inline internal/identity/manager/manager.go:215]:
```suggestion
			msg += "(https://www.pomerium.com/docs/reference/identity-provider-refresh-directory-settings)"
```

File: internal/identity/oauth/github/github.go [hunk 6]
```diff
@@ -1,6 +1,6 @@
 // Package github implements OAuth2 based authentication for github
 //
-// https://www.pomerium.io/docs/identity-providers/github.html
+// https://www.pomerium.com/docs/identity-providers/github.html
```
Comment (alexfornuto) [inline internal/identity/oauth/github/github.go:3]:
```suggestion
// https://www.pomerium.com/docs/identity-providers/github
```

File: internal/identity/oidc/auth0/auth0.go [hunk 7]
```diff
@@ -1,6 +1,6 @@
 // Package auth0 implements OpenID Connect for auth0
 //
-// https://www.pomerium.io/docs/identity-providers/auth0.html
+// https://www.pomerium.com/docs/identity-providers/auth0.html
```
Comment (alexfornuto) [inline internal/identity/oidc/auth0/auth0.go:3]:
```suggestion
// https://www.pomerium.com/docs/identity-providers/auth0
```

File: internal/identity/oidc/azure/microsoft.go [hunk 8]
```diff
@@ -1,6 +1,6 @@
 // Package azure implements OpenID Connect for Microsoft Azure
 //
-// https://www.pomerium.io/docs/identity-providers/azure.html
+// https://www.pomerium.com/docs/identity-providers/azure.html
```
Comment (alexfornuto) [inline internal/identity/oidc/azure/microsoft.go:3]:
```suggestion
// https://www.pomerium.com/docs/identity-providers/azure
```

File: internal/identity/oidc/gitlab/gitlab.go [hunk 9]
```diff
@@ -1,6 +1,6 @@
 // Package gitlab implements OpenID Connect for Gitlab
 //
-// https://www.pomerium.io/docs/identity-providers/gitlab.html
+// https://www.pomerium.com/docs/identity-providers/gitlab.html
```
Comment (alexfornuto) [inline internal/identity/oidc/gitlab/gitlab.go:3]:
```suggestion
// https://www.pomerium.com/docs/identity-providers/gitlab
```

File: internal/identity/oidc/google/google.go [hunk 10]
```diff
@@ -1,6 +1,6 @@
 // Package google implements OpenID Connect for Google and GSuite.
 //
-// https://www.pomerium.io/docs/identity-providers/google.html
+// https://www.pomerium.com/docs/identity-providers/google.html
```
Comment (alexfornuto) [inline internal/identity/oidc/google/google.go:3]:
```suggestion
// https://www.pomerium.com/docs/identity-providers/google
```

File: internal/identity/oidc/okta/okta.go [hunk 11]
```diff
@@ -1,6 +1,6 @@
 // Package okta implements OpenID Connect for okta
 //
-// https://www.pomerium.io/docs/identity-providers/okta.html
+// https://www.pomerium.com/docs/identity-providers/okta.html
```
Comment (alexfornuto) [inline internal/identity/oidc/okta/okta.go:3]:
```suggestion
// https://www.pomerium.com/docs/identity-providers/okta
```

File: internal/identity/oidc/ping/ping.go [hunk 12]
```diff
@@ -1,6 +1,6 @@
 // Package ping implements OpenID Connect for Ping
 //
-// https://www.pomerium.io/docs/identity-providers/ping.html
+// https://www.pomerium.com/docs/identity-providers/ping.html
```
Comment (alexfornuto) [inline internal/identity/oidc/ping/ping.go:3]:
```suggestion
// https://www.pomerium.com/docs/identity-providers/ping
```

**Review by alexfornuto — APPROVED** [review]

**Comment by desimone** [comment]
Suggested edit:
```diff
diff --git a/.github/workflows/test.yaml b/.github/workflows/test.yaml
index 361b2617..1ecdd172 100644
--- a/.github/workflows/test.yaml
+++ b/.github/workflows/test.yaml
@@ -49,7 +49,7 @@ jobs:
       - name: Lint
         if: runner.os == 'Linux'
         run: make lint
-      - name: test
+      - name: test2
         if: runner.os != 'Linux'
         run: make test
 

```

**Pull request merged**